src/e-cherry-pick.js

```diff
@@ -8,7 +8,6 @@ const path = require('path');
 const os = require('os');
 const { Octokit } = require('@octokit/rest');
 
-const { getCveForBugNr } = require('./utils/crbug');
 const { getGitHubAuthToken } = require('./utils/github-auth');
```

Comment:
```suggestion
const { getGerritPatchDetailsFromURL } = require('./utils/gerrit');
const { getGitHubAuthToken } = require('./utils/github-auth');
```

Reply:
Good catch! Forgot this during the refactor.